keyboards/cherrybstudio/cb1800/info.json

```diff
@@ -0,0 +1,15 @@
+{
+    "keyboard_name": "cb1800",
+    "url": "https://discord.gg/qVwv3gcq83",
+    "maintainer": "duongaanh",
+    "width": 19.5,
+    "height": 7.25,
+    "layouts": {
+        "LAYOUT_winkey": {
```

Comment:
This needs to match layout macros in the h file (eg, `LAYOUT_all`)

Reply:
oh, thanks. But I have a question that my PCB got so many configurable layout, do I need to create every layout combination for the info.json? it's will make so many layouts.